modules/www/main.tf

```diff
@@ -1,5 +1,18 @@
 # nerves_hub_www
 
+locals {
+  app_name = "nerves_hub_www"
+  ssm_prefix = "nerves_hub_www"
+
+  ecs_shared_env_vars = <<EOF
+    { "name" : "ENVIRONMENT", "value" : "${terraform.workspace}" },
+    { "name" : "APP_NAME", "value" : "${local.app_name}" },
+    { "name" : "HOST", "value" : "${var.host_name}" },
```

Comment:
Looks like this module only has `ENVIRONMENT` and `APP_NAME` in the task def originally. Do we need `HOST` and `CLUSTER` too?

Reply:
It threw an error about unset cluster and host and would not run without these...so, best answer, idk 🤷‍♀️ 

But it did not like it without these

Reply:
No, it was when the tasks would run. I could see it in the logs when the container would be starting up.

Reply:
The errors were [here](https://app.datadoghq.com/logs?query=host%3Anerves-hub-dev%20service%3Acloudwatch%20%40aws.awslogs.logGroup%3Anerves-hub-dev&agg_q=status%2Cservice&context_event=AQAAAYBMV0EizuFH3QAAAABBWUJNVjBUYUFBRFk0Sy1sU1plWW9BQUk&event=AQAAAYBMV0EizuFH4gAAAABBWUJNVjBUYUFBRFk0Sy1sU1plWW9BQU4&index=%2A&sort_m=%2C&sort_t=%2C&stream_sort=desc&to_event=AQAAAYBMV0GJf6JkwAAAAABBWUJNVjBQVkFBQkdCajBXRFVsay1RQVo&top_n=10%2C10&top_o=top%2Ctop&viz=&from_ts=1650542195110&to_ts=1650548228489&live=false) and [here](https://app.datadoghq.com/logs?query=host%3Anerves-hub-dev%20service%3Acloudwatch%20%40aws.awslogs.logGroup%3Anerves-hub-dev&agg_q=status%2Cservice&context_event=AQAAAYBMV0EizuFH3QAAAABBWUJNVjBUYUFBRFk0Sy1sU1plWW9BQUk&event=AQAAAYBMVwYm0pjr7wAAAABBWUJNVndxT0FBQS1rMVo1SnNNNS13QUs&index=%2A&sort_m=%2C&sort_t=%2C&stream_sort=desc&top_n=10%2C10&top_o=top%2Ctop&viz=&from_ts=1650542195110&to_ts=1650548228489&live=false)

Reply:
This is now receiving the KMS key and the Datadog key as inputs.

Reply:
Nice! I think in that case we should try going back to just the two env vars we had originally since the app should pull the env vars at runtime from ssm itself.

Reply:
Done ✅